pyfu_usb/__init__.py

```diff
@@ -153,7 +178,8 @@ def _dfu_download(
         xfer_size: Transfer size to use when downloading.
     """
     # Download data
-    with Progress() as progress:
+    progress = Progress()
```

Comment:
Started getting a weird `mypy` error and this change fixed it. 🤷🏻

Reply:
A win is a win